cypress/e2e/loginForm.cy.js

```diff
@@ -0,0 +1,78 @@
+//Login with  valid credentials
+describe("Login Functionality", () => {
+  it("allows the user to log in with valid credentials", () => {
+    cy.visit("http://127.0.0.1:5500/");
```

Comment:
```suggestion
    cy.visit("/");
```

Cypress runs it's own server